lib/dynamodb-integration.js

```diff
@@ -80,7 +80,7 @@ lib.upload = function(dynamodbClient, tableName, streamName, logEvents, options,
 
     debug('send to aws');
 
-    dynamodbClient.send(new BatchWriteItemCommand(payload), function (data, err) {
+    dynamodbClient.send(new BatchWriteItemCommand(payload), function (err, data) {
```

Comment:
you said

> we upgraded this package to v3 with the wrong order of parameters

so here you BOTH changed version to 2.0.2 AND changed the parameters order?

Reply:
I meant 2.0.1
thanks for clarifying